dev/src/backoff.ts

```diff
@@ -62,7 +62,8 @@ let delayExecution: typeof setTimeout = setTimeout;
  * @private
  * @param {function} handler A handler than matches the API of `setTimeout()`.
  */
-export function setTimeoutHandler(handler: typeof setTimeout): void {
+export function setTimeoutHandler(
```

Comment:
The usage in test/watch.ts suddenly didn't compile anymore since `setImmediate` is not of type `setTimeout`.